src/Symfony/Component/Security/Core/Encoder/Argon2iPasswordEncoder.php

```diff
@@ -42,7 +42,7 @@ public function encodePassword($raw, $salt)
             throw new BadCredentialsException('Invalid password.');
         }
 
-        if (\PHP_VERSION_ID >= 70200 && \defined('PASSWORD_ARGON2I')) {
```

Comment:
why is that dead code? it's not the same thing

Reply:
right, reverted